scripts/infra

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GT_SITE_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0") COMMAND OPTION[S]
+Execute Terraform subcommands with remote state management.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+        exit 0
+    fi
+
+    if [[ -n "${GT_SITE_SETTINGS_BUCKET}" ]]; then
+        pushd "${DIR}/../terraform"
+
+        case "${1}" in
+            plan)
+                rm -rf .terraform terraform.tfstate*
+                terraform init \
+                          -backend-config="bucket=${GT_SITE_SETTINGS_BUCKET}" \
+                          -backend-config="key=terraform/state"
+                terraform get -update
```

Comment:
This command can be removed, since `terraform init` takes care of updating modules.